data/data_5bus_pu.jl

```diff
@@ -913,6 +913,11 @@ ORDC_cost_ts = [
     TimeSeries.TimeArray(DayAhead + Day(1), repeat([ORDC_cost], 24)),
 ]
 
+hybrid_cost_ts = [
```

Comment:
Lets try to make this values reasonable or add a comment about why this value

Reply:
I'm not sure what value is sensible, I did change it to my best guess and also added a comment